validate/src/message.rs

```diff
@@ -203,7 +203,20 @@ pub fn embeds(embeds: &[Embed]) -> Result<(), MessageValidationError> {
             source: None,
         })
     } else {
+        let mut total = 0;
         for (idx, embed) in embeds.iter().enumerate() {
+            total += crate::embed::length(embed);
+
+            if total > crate::embed::EMBED_TOTAL_LENGTH {
```

Comment:
we should import this const i believe. I'll have to go over in another pr and unify the import style in unchanged code